### PR TITLE
CED-969-update-LFC-urls-es-nav-bar

### PR DESCRIPTION
**Pull request by suleiy**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-969

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Update LFC links in EsNavBar to new URLs according to[ site migration sheet](https://docs.google.com/spreadsheets/d/1-A4om_fArblc8s5sLA7XbufWchbh4mtF2om_biABEgc/edit#gid=212067427)
- changed the news urls to www.energysage.com/blog but the content of the link might not be ready

### 🥼 Testing

- manual testing

#### 🧐 Feedback Requested / Focus Areas

- Overall

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

File: es-vue-base/src/lib-utils/nav-bar-global-content.js [hunk 1]
```diff
@@ -441,17 +443,17 @@ export default (
...
                     name: 'Solar for nonprofit organizations',
-                    link: `${NEWS_DOMAIN}/solar-nonprofit-benefits-financing/`,
+                    link: `${ES_DOMAIN}/business-solutions/solar-nonprofit-benefits-financing/`,
                     newTab: true,
                 },
                 {
                     name: 'Energy storage for businesses',
-                    link: `${NEWS_DOMAIN}/energy-storage-for-businesses/`,
+                    link: `${ES_DOMAIN}/business-solutions/energy-storage-for-businesses/`,
                     newTab: true,
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-utils/nav-bar-global-content.js:457]: Anything that was pointing to `NEWS_DOMAIN` before and has `newTab` set to `true` should remove the `newTab: true` line, effectively setting it to `false`, now that everything is on the same domain (and, importantly, has the same navigation).

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Nice start! A few comments.

**Comment by suleiy** [comment]
Hi @ericdouglaspratt, thanks for the review! I've made the changes

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Looks good! Just one small thing.

File: es-vue-base/src/lib-utils/nav-bar-global-content.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-unused-vars */
 import { NAV_BAR_ICONS } from './nav-bar-constants';
 
 export default (
     ES_DOMAIN = process.env.ES_DOMAIN || 'https://www.energysage.com',
     // exposed separately from ES_DOMAIN in case we want to use prod for images in local/dev
     ES_IMAGE_DOMAIN = process.env.ES_IMAGE_DOMAIN || 'https://www.energysage.com',
-    NEWS_DOMAIN = process.env.NEWS_DOMAIN || 'https://news.energysage.com',
+    // keeping NEWS_DOMAIN to minimize breaking change, but it is deprecated
+    NEWS_DOMAIN = process.env.NEWS_DOMAIN || 'https://energysage.com',
```
[the remark this answers is not in the document]
Comment (suleiy) [reply es-vue-base/src/lib-utils/nav-bar-global-content.js]: fixed, thank you!

**Review by ericdouglaspratt — APPROVED** [review]
Nice!

**Pull request merged**